Source/WebCore/platform/graphics/ImageBuffer.cpp

```diff
@@ -478,6 +478,12 @@ void ImageBuffer::setVolatilityState(VolatilityState volatilityState)
         backend->setVolatilityState(volatilityState);
 }
 
+void ImageBuffer::clearContents()
+{
+    if (auto* backend = ensureBackendCreated())
+        backend->clearContents();
```

Comment:
Is it intentional that this instantiates `m_backend` if it wasn't already created?

Reply:
Hmm, in this case it's true that that wouldn't make sense, but if you had a bitmap-backed surface and were implementing "clear contents" surely you'd want to make the backend in order to do so?

Reply:
Ah, right — that makes sense!